### DISCUSSION TITLE
Product features name - differ between display and internal name

### DISCUSSION THREAD

**Discussion by Flying--Dutchman** [PrestaShop/PrestaShop#40250] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

Currently when adding custom features, the given name is used in the front- and backoffice. 
When dealing with completely different products you can end up with something like:

- Feature: Material
- Values; Gold, Silver, Steel

---
- Feature: Material
- Values: Glass, Crystal, Lead Crystal.

### Describe the solution you'd like

When adding a new product, you would always have to look for the correct feature. 
Ideally we would have something like:

- Feature: Material-For-Cat-XX
- Display name: Material
- Values; Gold, Silver, Steel

---
- Feature: Material-For-Cat-XY
- Display name: Material
- Values: Glass, Crystal, Lead Crystal.

![image](https://user-images.githubusercontent.com/23308427/182442397-7a638331-8d3f-42a9-974b-688277d61e59.png)

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by hibatallahAouadni** [comment]
Hello @Flying--Dutchman 

Could you please explain more what's your sugguestion?
Waiting for your feedback.

Thanks!

**Comment by Flying--Dutchman** [comment]
Sure. Basically the idea is to add a new field to the "Product feature" form, called "Display name". The value from that field would be used in the frontoffice (what the customer would see), while the normal "name" field would be used in the backoffice. 

Especially when working with addons this would come very handy, eg. mapping fields for eBay sync.

![image](https://user-images.githubusercontent.com/9158539/182252029-6994e0b9-ed66-4301-93e0-002094bd9d28.png)

**Comment by hibatallahAouadni** [comment]
Hello @Flying--Dutchman 

I just edit your issue description following your feedback.
Thank you for your suggestion. The Maintainer Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you
